src/Stratis.Bitcoin.Features.Wallet/Models/RequestModels.cs

```diff
@@ -187,6 +200,26 @@ public class BuildTransactionRequest : TxFeeEstimateRequest
         public string Password { get; set; }
 
         public string OpReturnData { get; set; }
+
+        /// <inheritdoc />
+        public IEnumerable<ValidationResult> Validate(ValidationContext validationContext)
+        {
+            if (!string.IsNullOrEmpty(this.FeeAmount) && !string.IsNullOrEmpty(this.FeeType))
+            {
+                yield return new ValidationResult(
+                    "The query parameters 'feeamount' and 'feetype' cannot be set at the same time. " +
```

Comment:
Replace `feeamount` and `feetype` with `nameof(this.FeeAmount)` and `nameof(this.FeeType)`?